extensions-core/druid-pac4j/pom.xml

```diff
@@ -38,7 +38,7 @@
 
     <!-- Following must be updated along with any updates to pac4j version. One can find the compatible version of nimbus libraries in org.pac4j:pac4j-oidc dependencies-->
     <nimbus.lang.tag.version>1.7</nimbus.lang.tag.version>
-    <nimbus.jose.jwt.version>8.22.1</nimbus.jose.jwt.version>
+    <nimbus.jose.jwt.version>9.37.2</nimbus.jose.jwt.version>
```

Comment:
is it compatible with pac4j-oidc dependency currently in use?

Reply:
Cannot infer compatibility as the maven page just lists the dependency for a specified version -- which was the earlier version. I've updated `pac4j-oidc` also to `6.0.1` which lists `9.37.3` as its dependency, and the version of `oauth2.oidc.sdk.version` corresponding to it as well.

Reply:
Tried multiple versions of pac4j (6.0.1, 5.7.3, 5.3.1) but all are failing compilation, so reverting it to the original version.